decidim-core/app/views/decidim/homepage/show.html.erb

```diff
@@ -10,5 +10,6 @@
 <% Decidim::ContentBlock.published.for_scope(:homepage, organization: current_organization).each do |content_block| %>
   <% next unless content_block.manifest %>
 
+  <a id="<%= content_block.manifest.name %>"></a>
```

Comment:
Should we add a spec for checking this one? I know it can be overkill, but as we're starting the redesign, I'm afraid this could be lost in the near future.

Reply:
@andreslucena the spec has been added.